docs/getting_started.md

```diff
@@ -133,6 +139,35 @@ for result in results:
     print(f'{result[0]}: ', result[1])
 ```
 
+Here is an example of building the model and testing with a given rawframes directory.
+
+```python
+import torch
+
+from mmaction.apis import init_recognizer, inference_recognizer
+
+config_file = 'configs/recognition/tsn/tsn_r50_inference_1x1x3_100e_kinetics400_rgb.py'
+# download the checkpoint from model zoo and put it in `checkpoints/`
+checkpoint_file = 'checkpoints/tsn_r50_1x1x3_100e_kinetics400_rgb_20200614-e508be42.pth'
+
+# assign the desired device.
+device = 'cuda:0' # or 'cpu'
+device = torch.device(device)
+
+ # build the model from a config file and a checkpoint file
+model = init_recognizer(config_file, checkpoint_file, device=device, use_frames=True)
+
+# test rawframe directory of a single video and show the result:
+video = 'SOME_DIR_PATH/'
```

Comment:
what's the structure of this dir, "SOME_DIR_PATH/video_name/img_xxxx.jpg" or "SOME_DIR_PATH/img_xxxx.jpg"?

Reply:
* When filename is "SOME_DIR_PATH/video_name/img_xxxx.jpg" and `data_prefix` is None in config file. `video = SOME_DIR_PATH/video_name`.
* When filename is "SOME_DIR_PATH/video_name/img_xxxx.jpg" and `data_prefix` is "SOME_DIR_PATH" in config file. `video = video_name`.
* When filename is "SOME_DIR_PATH/img_xxxx.jpg" and `data_prefix` is None in config file. `video = SOME_DIR_PATH`.

Reply:
As a default, we set `data_prefix=None` in config file.

Reply:
is this info clear to reader?

Reply:
Add the description in a comment to make this info clear to readers.